packages/docs/src/lang/en/components/Forms.json

```diff
@@ -23,6 +23,7 @@
     }
   },
   "props": {
+    "disabled": "If enabled this will set all child inputs to disabled",
```

Comment:
Current message ("if enabled ... to disabled") sounds weird to me, maybe "If set to `true`" or just "Disables all child inputs"